ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -8187,6 +8188,27 @@ static void unwindAddFiles(HqlExprArray & args, IHqlExpression * expr, bool isOr
         args.append(*LINK(expr));
 }
 
+static IHqlExpression * queryRootConcatActivity(IHqlExpression * expr)
+{
+    loop
+    {
+        node_operator curOp = expr->getOperator();
+        switch (curOp)
+        {
+        case no_nofold:
+        case no_section:
+        case no_sectioninput:
+        case no_preservemeta:
+        case no_nocombine:
+        case no_forcegraph:
```

Comment:
no_forcegraph wasn't previously part of this logic, should there be a test added to the new test file to reflect this?